plugin/src/withPodfile.ts

```diff
@@ -19,65 +20,68 @@ export const withPodfile: ConfigPlugin<{
 
       const useExpoModules =
         excludedPackages && excludedPackages.length > 0
-          ? `exclude = ["${excludedPackages.join(`", "`)}"]\n  use_expo_modules!(exclude: exclude)`
+          ? `exclude = ["${excludedPackages.join(`", "`)}"]\n    use_expo_modules!(exclude: exclude)`
           : "use_expo_modules!";
 
       const appClipTarget = `
-target '${targetName}' do
-  ${useExpoModules}
+  target '${targetName}' do
```

Comment:
`mergeContents` don't have intent config, if I will have some time I will try to add it to expo, but for now we just need to add indent in the string that will be added to podfile.